takeshape/penny
# use seo title and description for product and collection pages

### Test Plan (Steps to test):

1. Make some changes to the SEO title and description on a product and on a collection in Shopify
1. Pull this branch
2. Run `npm run dev`
3. Inspect the rendered DOM in your inspector tool, ensure that your special SEO info is preset
